src/storage-blob-preview/setup.py

```diff
@@ -16,7 +16,7 @@
 
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-VERSION = '0.5.1'
+VERSION = '0.5.2'
```

Comment:
Upgrade version here and release note is already updated in https://github.com/Azure/azure-cli-extensions/blob/main/src/storage-blob-preview/HISTORY.rst#052